src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -84,4 +84,13 @@ public static void checkIsEmpty(String toTest, String qualifiedFieldName, Errors
             ApiLogger.infoContext(loggingContext, qualifiedFieldName + " Field should not be populated");
         }
     }
+
+    public static boolean checkIsNotEqual(String string1, String string2, String errorMsg, String qualifiedFieldName, Errors errors, String loggingContext) {
```

Comment:
Needs tests added to StringValidatorsTest

Reply:
also needs to null check the string if not using StringUtils.equals. null pointer will occur if something passes string1 as null